BlueDBOnDisk/src/main/java/org/bluedb/disk/ReadableDbOnDisk.java

```diff
@@ -80,7 +85,7 @@
 	public <V extends Serializable> ReadableBlueTimeCollection<V> getTimeCollection(String name, Class<V> valueType) throws BlueDbException {
 		try {
 			ReadableBlueCollection<V> collection = getExistingCollection(name, valueType);
-			if(collection instanceof ReadableBlueTimeCollection) {
+			if (collection instanceof ReadableBlueTimeCollection) {
```

Comment:
There are a bunch of meaningless changes that just update formatting and whitespace. They make it harder to see the recent meaningful history of these lines, they make it harder to look through the PR for meaningful changes, and they can sometimes result in unnecessary merge conflicts. I don't mind formatting changes if you're updating a block of code, but if you could avoid updating the formatting of methods that you aren't changing that would be helpful.